### PR TITLE
Manually reload plugins

### PR DESCRIPTION
**Pull request by petlofg**
* Allow users to manually reload of plugins in the language server via a new command in the VSCode client, for example if changing the version of a plugin.
* Expanded and corrected sample project.
* Fixed bug in language server that enabled Ion lookups within T4 tags if they start on first column.

### CODE REVIEW

File: Samples/Ion/Ion/Ion.dafproj [hunk 1]
```diff
@@ -17,6 +17,10 @@
 		<csCompile Include="**\*.cs" />
 		<sqlFile Include="**\*.sql" Exclude="$(OutputPath)\**\*.sql" />
 	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="Daf.Core.Sql" Version="0.0.4" />
+		<PackageReference Include="Daf.Core.Ssis" Version="0.0.5" />
```
Comment (bjorhn) [inline Samples/Ion/Ion/Ion.dafproj:22]: Should these be wildcard includes? 0.0.* perhaps?

File: Daf.Core.VSCode/src/extension.ts [hunk 2]
```diff
@@ -224,6 +238,15 @@ async function createUserFile(): Promise<void>
 	});
 }
 
+async function reloadPlugins(): Promise<void>
+{
+	if (lspClient != undefined){
+		lspClient.sendRequest('daf/reloadplugins');
+	} else {
+		vscode.window.showInformationMessage('Cannot reload Daf plugins, language server has not been initiated.');
+	}
```
Comment (bjorhn) [inline Daf.Core.VSCode/src/extension.ts]: For the most part it appears that we put braces on newlines in this document, but not always.. I wonder if we should decide on a standard and stick to it? I know it's more common to put it in the same brace in typescript, so maybe we should stick with that. It's just so awful to look at...

Doesn't have to be solved for this PR, but maybe we should discuss it and create an issue.

**Review by bjorhn — CHANGES_REQUESTED** [review]

File: Samples/Ion/Ion/Ion.dafproj (hunk 1 above)
Comment (petlofg) [reply Samples/Ion/Ion/Ion.dafproj:22]: My thinking here was that it should always be possible to run the samples, though I guess we would not introduce API breaking changes when bumping the minor version. Having the version completely fixed means we don't have to worry about it though.

File: Daf.Core.VSCode/src/extension.ts (hunk 2 above)
Comment (petlofg) [reply Daf.Core.VSCode/src/extension.ts]: I'd rate following the most common way of writing it higher than personal preference, though this seems to be the only place where it's written at the same line. I'll change it to be similar to the other code in the file for now, then we can discuss how we should format it in a separate issue.

Comment (bjorhn) [reply Daf.Core.VSCode/src/extension.ts]: Agreed.

File: Samples/Ion/Ion/Ion.dafproj (hunk 1 above)
Comment (bjorhn) [reply Samples/Ion/Ion/Ion.dafproj:22]: Fair, let's go with that. :)

**Review by bjorhn — APPROVED** [review]

**Pull request merged**